includes/managers/icons.php

```diff
@@ -502,12 +502,13 @@ public function register_admin_tools_settings( Tools $settings ) {
 			},
 			'fields' => [
 				[
-					'label'      => esc_html__( 'Font Awesome Upgrade', 'elementor' ),
+					'label' => esc_html__( 'Font Awesome Upgrade', 'elementor' ),
 					'field_args' => [
 						'type' => 'raw_html',
-						'html' => sprintf( '<span data-action="%s" data-_nonce="%s" class="button" id="elementor_upgrade_fa_button">%s</span>',
+						'html' => sprintf( '<span data-action="%s" data-_nonce="%s" data-redirect-url="%s" class="button" id="elementor_upgrade_fa_button">%s</span>',
```

Comment:
it's not a "redirect URL" anymore. Maybe change the name to something else that suggests it's a document ID?

Reply:
Done